platform/src/pages/analytics/tabs/OverView.jsx

```diff
@@ -33,6 +33,24 @@ const OverView = () => {
     }
   }, [chartDataRange, sites]);
 
+  const dummyData = {
+    siteDetails: {
+      search_name: 'N/A',
```

Comment:
Hi @OchiengPaul442 , thanks for this. Just think that we might want to rethink this.
lf there is no data, should it also mean that the location name should not exist? Or the location name must exist irrespective of whether the data is present or not?

Reply:
Hey @Baalmart, 
Thanks for your Feedback, I also thought the same thing but for every site selected it always returns an empty array of data so am not quite sure yet if it is just staging that's lucking the data but further testing is still ongoing.

NB: Also, the reason why the lower charts have data and the cards don't is because separate endpoints have been used in both cases.

Currently, the measurements/recent endpoint returns an empty array of measurements, causing the cards to not display data. To address this, the current implementation displays data from user-selected sites if measurements are available. If fewer than the expected number of sites have measurements, dummy data is used to fill the remaining card spaces. 

NB: An alternative approach could be to display random sites that have not been selected by the user to fill the empty spaces if the user-selected sites lack measurements.